### DISCUSSION TITLE
Possibility to add attributes with mask

### DISCUSSION THREAD

**Discussion by AlessandroAmos** [lunarphp/lunar#1014] [Feature Requests]
possibility to create a text attribute with mask to create fields like this:
![image](https://user-images.githubusercontent.com/118480471/203110666-e412a98b-bc3f-4157-bc0b-ad79a6832792.png)

**Comment by glennjacobs** [comment]
Although this can be achieved with HTML5 with patterns and placeholders, I prefer it when it automatically formats what you type in and that will likely require some JS.

Anyone know of a good library which could be used?

**Comment by AlessandroAmos** [comment]
I think it would be interesting to leave 2 fields, a text field where I can fill in the mask... but still allow someone more experienced to place the mask configuration js object

https://github.com/RobinHerbots/Inputmask

**Comment by alecritson** [comment]
I’ve always quite liked https://nosir.github.io/cleave.js/

**Comment by AlessandroAmos** [comment]
> I’ve always quite liked https://nosir.github.io/cleave.js/

I think it would be difficult to configure the mask through the panel using this package
